openshift/02-apicast-builder-imagestream.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  name: apicast
+  labels:
+    app: apicast
+spec:
+  tags:
+  - name: builder
```

Comment:
Maybe add the branch name to the tag name? So we know it is "master" and not some tag?

Reply:
Good Idea 👍